mitodl/micromasters
# Use poetry for python dependency management

# What are the relevant tickets?
Fixes https://github.com/mitodl/micromasters/issues/5343
# Description (What does it do?)
Use poetry for python dependency management.
Removes requirements files.

# How can this be tested?
Nothin should break.
Run `docker-compose build --no-cache web celery`
CI tests should pass.
